### DISCUSSION TITLE
How to get the place name based on current use location coords

### DISCUSSION THREAD

**Discussion by prox2** [rnmapbox/maps#1229] [Q&A] [answered]
I want to display the name or place related to specific coords as in  

```
<MapboxGL.UserLocation
                        visible={true}
                        onUpdate={this.onUserLocationUpdate}

                    />
```

it retrieve  the coords only how to retrieve the name of place that these coords related to

**Comment by ferdicus** [comment, marked as answer]
Something like this doesn't happen out of the box. 
What you want is `reverse geocoding`: https://docs.mapbox.com/api/search/geocoding/#reverse-geocoding
